src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+type PrepareTodo = {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number
+  user?: {
```

Comment:
You need user interface ( duplicate )
You don't need possible value. The user needs to be in todo.